go.mod

```diff
@@ -31,6 +31,7 @@ require (
 require (
 	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
 	github.com/Microsoft/go-winio v0.6.0 // indirect
+	github.com/ajg/form v0.0.0-20160822230020-523a5da1a92f // indirect
```

Comment:
Leaving this as a reminder, before merging this MR we need a `go get github.com/tsuru/tsuru@latest` after https://github.com/tsuru/tsuru/pull/2703 is merged.